packages/react/src/components/Tabs/TabPanel.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+interface TabPanelProps {
```

Comment:
```suggestion
interface TabPanelProps extends React.HTMLAttributes<HTMLDivElement> {
```

This will allow any consumer to add additional props to the panel.